.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release Helm Charts
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    env:
+      HELM_VERSION: v3.5.2
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Fetch history
+        run: git fetch --prune --unshallow
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: ${{ env.HELM_VERSION }}
+
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.2.0
+        env:
+          CR_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          CR_PAGES_BRANCH: main
```

Comment:
Is this variable used?

Reply:
Yes, 
When use the `main` branch on github pages, the default branch in `chart-releaser` is `gh-pages`, so need to specify `main` branch.

https://github.com/helm/chart-releaser/blob/ceb402678bdb55890bf00ade68c34f90cc76d027/cr/cmd/index.go#L60

Reply:
Thanks. 👍
I didn't know that the command-line options can also be set by environment variables.
https://github.com/helm/chart-releaser/blob/main/README.md#configuration
> All command-line flags can also be set via environment variables or config file. Environment variables must be prefixed with CR_. Underscores must be used instead of hyphens